### PR TITLE
docs(): update chain-id from mocha to mocha-4

### PR DESCRIPTION
**Pull request by packetstracer**
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: docs/nodes/validator-node.mdx [hunk 1]
```diff
@@ -17,10 +17,10 @@ Validator nodes allow you to participate in consensus in the Celestia network.
 The following hardware minimum requirements are recommended for running the
 validator node:
 
-* Memory: **8 GB RAM**
-* CPU: **6 cores**
-* Disk: **500 GB SSD Storage**
-* Bandwidth: **1 Gbps for Download/1 Gbps for Upload**
+- Memory: **8 GB RAM**
```
Comment (jcstein) [inline docs/nodes/validator-node.mdx:20]:
```suggestion
* Memory: **8 GB RAM**
```

File: docs/nodes/validator-node.mdx [hunk 2]
```diff
@@ -17,10 +17,10 @@ Validator nodes allow you to participate in consensus in the Celestia network.
...
+- CPU: **6 cores**
```
Comment (jcstein) [inline docs/nodes/validator-node.mdx:21]:
```suggestion
* CPU: **6 cores**
```

File: docs/nodes/validator-node.mdx [hunk 3]
```diff
@@ -17,10 +17,10 @@ Validator nodes allow you to participate in consensus in the Celestia network.
...
+- Disk: **500 GB SSD Storage**
```
Comment (jcstein) [inline docs/nodes/validator-node.mdx:22]:
```suggestion
* Disk: **500 GB SSD Storage**
```

File: docs/nodes/validator-node.mdx [hunk 4]
```diff
@@ -17,10 +17,10 @@ Validator nodes allow you to participate in consensus in the Celestia network.
...
+- Bandwidth: **1 Gbps for Download/1 Gbps for Upload**
```
Comment (jcstein) [inline docs/nodes/validator-node.mdx:23]:
```suggestion
* Bandwidth: **1 Gbps for Download/1 Gbps for Upload**
```

File: docs/nodes/validator-node.mdx (hunk 1 above)
Comment (jcstein) [reply docs/nodes/validator-node.mdx:20]: will cause lint error if not *

**Comment by jcstein** [comment]
thank you for this PR!

**Comment by packetstracer** [comment]
you're welcome, glad to help

**Pull request merged**